### DISCUSSION TITLE
Rubberband selection

### DISCUSSION THREAD

**Discussion by 9Skrip** [doublecmd/doublecmd#1990] [Suggestions]
Please add an option to use mouse Rubberband selection like TC has
![изображение](https://github.com/user-attachments/assets/73ba810e-65b3-4508-997b-b8ffd80a211a)

**Comment by advice2020** [comment]
I know this one has been asked before by other users, this was something that I wanted to mention in the past as well but was holding off because of the often mentioned keyboard centric use of the file manager, but I agree this would great for certain scenarios.
Would also work great with the recent feature of drag and drop moves.
If added, would of course be an option, not default setting.

DC offers this to be setup with the RMB which works very smoothly but RMB is not desired.  Also requires holding down RMB open context menus.  Both amazingly programmed, but because it is not standard behavior I avoid using this setting because do not want to get in the habit of performing these actions in this way with RMB.

I will share just a couple things about this, that maybe could be helpful.
The issue with the use of rubberband selection is the fact that dragging mouse on top of filenames is set to copying/moving files as it should be.  Although there are a few solutions I have seen to be able to apply rubberband action.

`IDEAS`
Can activate rubberband when dragging from.....
-Blank area to the right of folder/filenames within the "Name" column itself.
-Column(s) to the right of the "Name" column (meaning that all other columns other than "Name" would perform rubberband) 
-This one is my idea, I do not like it fully because the area to perform the rubberband action would be narrow and uncomfortable to apply if used regularly, but could expand the idea of how DC allow the clicking on MIME / File Type icons to select/mark items, could allow mouse to drag on them to achieve rubberband selection.  Again probably would not use this one only, but could be incorporated with one of the other two solutions mentioned above.
-Another one that I have seen that is really not desired/feels very akward to use, is where the rubber band always works and if user want to copy/move a single item they have to click once to highlight that item and then again to drag that item.  I have not seen this used in orthodox file managers because I am not sure the mark/select methods would work with it, but regardless hopefully this method would not be used :)

Anyways, sorry just wanted to share some ideas I was saving about this topic.
Thank You
